test/http-api/over-ipfs-api/bootstrap.js

```diff
@@ -28,6 +28,27 @@ module.exports = (ctl) => {
         })
       })
 
+      it('prevents duplicate inserts of bootstrap peers', () => {
+        return ctl
+          .bootstrap
+          .add(validIp4)
+          .then(res => {
+            expect(res).to.be.eql({ Peers: [validIp4] })
+            return ctl.bootstrap.add(validIp4)
+          })
+          .then((res) => {
+            expect(res).to.be.eql({ Peers: [validIp4] })
+            return ctl.bootstrap.list()
+          })
+          .then((res) => {
+            expect(res).to.exist()
+            const insertPosition = res.Peers.indexOf(validIp4)
+            expect(insertPosition).to.not.equal(-1)
+            const duplicatePosition = res.Peers.indexOf(validIp4, insertPosition + 1)
+            expect(duplicatePosition).to.equal(-1)
```

Comment:
I'm not getting your question @VictorBjelkholm, these test nodes have their bootstrap lists empty by default:

- https://github.com/ipfs/js-ipfs/blob/master/test/utils/ipfs-factory-instance/default-config.json#L39
- https://github.com/ipfs/js-ipfs/blob/master/test/utils/ipfs-factory-daemon/default-config.json#L48